src/main/java/teammates/storage/sqlentity/responses/FeedbackConstantSumResponse.java

```diff
@@ -22,6 +25,15 @@ protected FeedbackConstantSumResponse() {
         // required by Hibernate
     }
 
+    public FeedbackConstantSumResponse(
+            FeedbackQuestion feedbackQuestion, String giver,
+            Section giverSection, String receiver, Section receiverSection,
+            FeedbackResponseDetails responseDetails
+    ) {
+        super(feedbackQuestion, giver, giverSection, receiver, receiverSection);
+        this.setAnswer((FeedbackConstantSumResponseDetails) responseDetails);
+    }
+
```

Comment:
Please also this to `recipient`. Same for all the other feedback response classes.

Reply:
changed!